.changeset/strange-peas-confess.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Empty cause nobody needs to know, and it'll go out with the next version anyway 😄